### PR TITLE
chore(karpenter): upgrade karpenter to v0.32.6

### LINKED ISSUE #16311: Unable to configure disruption controls for karpenter
I am unable to figure out how to add a disruption consolidationPolicy and expireAfter in my karpenter node pools for kops. Where do I configure this?

The karpenter docs discuss this here.

https://karpenter.sh/v0.32/concepts/nodepools/#specdisruption

I'm not even able to see a CRD for karpenter NodePools, so I'm guessing kops has another way of managing the disruption controls?

```
  disruption:
    consolidationPolicy: WhenUnderutilized
    expireAfter: 720h # 30 * 24h = 720h
```

### PR DESCRIPTION
**Pull request by moshevayner**
**What this PR does / why we need it**:
Bumps karpenter version to 0.32.6

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
fixes #16311

**Special notes for your reviewer**:
[Release page](https://github.com/aws/karpenter-provider-aws/releases/tag/v0.32.6)

### CODE REVIEW

**Review by hakman — CHANGES_REQUESTED** [review]

File: upup/models/cloudup/resources/addons/karpenter.sh/k8s-1.19.yaml.template [hunk 1]
```diff
@@ -1640,23 +613,23 @@ spec:
           topologyKey: topology.kubernetes.io/zone
           whenUnsatisfiable: ScheduleAnyway
       tolerations:
-        - key: node-role.kubernetes.io/master
-          operator: Exists
-        - key: node-role.kubernetes.io/control-plane
-          operator: Exists
```
[the remark this answers is not in the document]
Comment (moshevayner) [reply upup/models/cloudup/resources/addons/karpenter.sh/k8s-1.19.yaml.template:1646]: Yeah, that makes sense. I must've overlooked those, and that could make sense as to why the tests are failing.
Let me give that a try.
Thanks for catching that!

**Comment by hakman** [comment]
I guess CRDs need updating:
```
I0208 05:04:17.581270    5067 addon.go:192] Applying update from "s3://k8s-kops-prow/e2e-pr16327.pull-kops-e2e-aws-karpenter.test-cncf-aws.k8s.io/addons/karpenter.sh/k8s-1.19.yaml"
I0208 05:04:17.581294    5067 s3fs.go:378] Reading file "s3://k8s-kops-prow/e2e-pr16327.pull-kops-e2e-aws-karpenter.test-cncf-aws.k8s.io/addons/karpenter.sh/k8s-1.19.yaml"
W0208 05:04:17.942636    5067 health.go:69] expected status.conditions to be list, got <nil>
W0208 05:04:17.989700    5067 health.go:69] expected status.conditions to be list, got <nil>
W0208 05:04:18.004385    5067 health.go:69] expected status.conditions to be list, got <nil>
W0208 05:04:18.037816    5067 health.go:69] expected status.conditions to be list, got <nil>
W0208 05:04:18.104261    5067 health.go:69] expected status.conditions to be list, got <nil>
W0208 05:04:18.185098    5067 health.go:69] expected status.conditions to be list, got <nil>
I0208 05:04:18.279025    5067 health.go:63] status conditions not found for PodDisruptionBudget.policy:kube-system/karpenter
I0208 05:04:18.478384    5067 health.go:63] status conditions not found for Secret:kube-system/karpenter-cert
I0208 05:04:19.883033    5067 health.go:63] status conditions not found for Service:kube-system/karpenter
I0208 05:04:19.982831    5067 health.go:63] status conditions not found for Deployment.apps:kube-system/karpenter
I0208 05:04:20.081517    5067 health.go:63] status conditions not found for MutatingWebhookConfiguration.admissionregistration.k8s.io:defaulting.webhook.karpenter.k8s.aws
I0208 05:04:20.178238    5067 health.go:63] status conditions not found for ValidatingWebhookConfiguration.admissionregistration.k8s.io:validation.webhook.karpenter.sh
I0208 05:04:20.277927    5067 health.go:63] status conditions not found for ValidatingWebhookConfiguration.admissionregistration.k8s.io:validation.webhook.config.karpenter.sh
I0208 05:04:20.376625    5067 health.go:63] status conditions not found for ValidatingWebhookConfiguration.admissionregistration.k8s.io:validation.webhook.karpenter.k8s.aws
W0208 05:04:20.376686    5067 results.go:63] error from apply on karpenter.k8s.aws/v1alpha1, Kind=AWSNodeTemplate /nodes: error getting rest mapping for karpenter.k8s.aws/v1alpha1, Kind=AWSNodeTemplate: no matches for kind "AWSNodeTemplate" in version "karpenter.k8s.aws/v1alpha1"
W0208 05:04:20.376713    5067 results.go:63] error from apply on karpenter.sh/v1beta1, Kind=NodePool /nodes: error getting rest mapping for karpenter.sh/v1beta1, Kind=NodePool: no matches for kind "NodePool" in version "karpenter.sh/v1beta1"
W0208 05:04:20.376746    5067 results.go:56] consistency error (healthy counts): &applyset.ApplyResults{total:30, applySuccessCount:28, applyFailCount:2, healthyCount:28, unhealthyCount:0}
```

**Comment by guanchzhou** [comment]
Under the lights of https://github.com/kubernetes/kops/pull/16758 is this PR still needed?

**Comment by hakman** [comment]
> Under the lights of #16758 is this PR still needed?

I think @rifelpet's PR is a more advanced version. Also, I think @moshevayner is pretty busy these days.
Closing for now.
/close

**Comment by moshevayner** [comment]
> > Under the lights of #16758 is this PR still needed?
> 
> I think @rifelpet's PR is a more advanced version. Also, I think @moshevayner is pretty busy these days. Closing for now. /close

Thanks @hakman !
Indeed I've been a bit off the grid lately.. Hoping to be able to get back on soon 🙏🏼

**Pull request closed without merging**